### ISSUE TITLE
Bootstrap jquery dependencies

### ISSUE THREAD

**Issue by nazjunaid** [jspm/registry#873]
Not sure if this is intentional or even a bug, but when I do...

```
jspm install jquery
```

It downloads the ```npm:jquery@^2.2.3``` package, however

```
jspm init bootstrap
```

Downloads jquery from ```github:components/jquery@^2.2.1``` as a dependency of ```github:twbs/bootstrap@^3.3.6```

It's a bit odd as I now have two versions of jQuery which I didn't expect. 

Using jspm 0.16

Sorry if I'm being a noob

**Comment by guybedford** [comment]
Not a noob question at all - this is a great point. Ideally jquery would be a peer dependency so that this conflict is flagged and resolved, but this support is only in the jspm 0.17 beta currently.

So when we switched to using npm:jquery as the main jquery version, these other dependencies also needed to switch over.

I've added https://github.com/jspm/registry/commit/52fcf0f0056f08592bb42babf0c81988ff180fe1 to alter this so it should work now.
